GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -169,6 +169,10 @@ @implementation GIDSignIn {
 
 #pragma mark - Public methods
 
+// Handles the custom URL returned from AppAuth or Device Policy App (EMM flow).
```

Comment:
I think you need three slashes: `///`.

Reply:
I tried this. I think this only works for comments in the header file. The comments in the source file would not be displayed.

Reply:
Gah, of course. What do you think about moving this to `GIDSignIn.h` and merging this information with the existing doc comment?

Reply:
This is the implementation details for us. Our clients should not know this.

Reply:
Since this is a public method, don't these comments apply more broadly than "implementation details" implies?

Reply:
We have the comments in the public API for the clients. They don't need to understand how we use this method in our implementation.

Reply:
Perhaps I'm not understanding things correctly, so please bear with me. :) 

My thinking is that this method is in the public API. For example, our [sample app](https://github.com/google/GoogleSignIn-iOS/blob/8345fcc43c38cefb67f1ad0d5dcd51a1d036e460/Samples/ObjC/SignInSample/Source/AppDelegate.m#L49) uses this method much the same way clients will. 

Therefore, it seems to me that these comments/details could relevant to others.

Reply:
Following up from an offline conversation...

It seems like there was already a discussion about this that led to these comments landing here. Is that right @petea? If so, I don't want to block this PR on a small detail. I'll approve and @Alex-4-Git you can decide what to do!

Reply:
Peter helped me improve the comments. We can put it in the source file.